scipy/stats/tests/test_distributions.py

```diff
@@ -4433,7 +4425,7 @@ def test_no_name_arg(self):
         stats.rv_discrete()
 
 
-def TestArgsreduce():
+def test_args_reduce():
```

Comment:
This test is currently failing. @tupui @mdhaber I'm not really sure what the correct behaviour is, do we still want this test?

Reply:
This looks very old and doesn't seem to belong in `test_distribution`. But please do a little digging as to what it is supposed to do and why it is failing. I don't know offhand.

Reply:
Looking closer the function it tests is never used and is dead code so can just be removed

Reply:
Turns out it is not dead, I was only searching `.py` files but it is used in the cython wrapper of unu.ran. More investigation required...